src/core/navbar/navbar.jsx

```diff
@@ -0,0 +1,16 @@
+import {AppBar, Toolbar, Typography} from '@material-ui/core'
+import MenuIcon from '@material-ui/icons/Menu'
+import {RestaurantMenu} from "@material-ui/icons";
+import {NavbarIcons} from "./navbar-icons/navbar-icons";
+import {NavbarTitle} from "./navbar-title/navbar-title";
+import AddShoppingCartIcon from '@material-ui/icons/AddShoppingCart';
+
+export const NavBar = () =>
+    <AppBar position ='static'>
+        <Toolbar variant ='dense'>
+            <NavbarIcons link={'/'} icon={<MenuIcon/>} />
```

Comment:
![image](https://user-images.githubusercontent.com/23569938/109488208-ea036300-7a8d-11eb-8e03-6e47500733ab.png)
It's maybe a small detail, but this icon is pretty misleading, since before pressing it I thought it's gonna open a menu, but it redirected me to the `/` route. Maybe choose something else?

Reply:
![image](https://user-images.githubusercontent.com/23569938/109488356-233bd300-7a8e-11eb-8d7d-113ec4feeee4.png)
Also is the application's title meant to be between your links? :)))

Reply:
Yes

Reply:
Also, the misleading icon has been changed.

Reply:
Okay